pkg/metrics/metrics.go

```diff
@@ -59,13 +65,31 @@ func NewRecorder() (*Recorder, error) {
 	}
 	r.repository = repository
 
+	status, err := tag.NewKey("status")
+	if err != nil {
+		return nil, err
+	}
+	r.status = status
+
+	reason, err := tag.NewKey("reason")
+	if err != nil {
+		return nil, err
+	}
+	r.reason = reason
+
 	err = view.Register(
 		&view.View{
 			Description: prCount.Description(),
 			Measure:     prCount,
 			Aggregation: view.Count(),
 			TagKeys:     []tag.Key{r.provider, r.eventType, r.namespace, r.repository},
 		},
+		&view.View{
+			Description: prDurationCount.Description(),
+			Measure:     prDurationCount,
+			Aggregation: view.Sum(),
+			TagKeys:     []tag.Key{r.namespace, r.repository, r.status, r.reason},
```

Comment:
@savitaashture console team only wants sum of duration taken by all pipelineruns, I asked about pipelinerun name tag and it isn't wanted by team, and also it will affect on cardinality of  metric if we put pipelinerun tag.

Reply:
I see okay
then its fine to not add